frontend-web/webclient/app/Admin/AppAauAdmin.tsx

```diff
@@ -15,15 +15,22 @@ import {bulkRequestOf} from "DefaultObjects";
 const AppAauAdmin: React.FunctionComponent = props => {
     const [commandLoading, invokeCommand] = useCloudCommand();
     const [job, fetchJob] = useCloudAPI<Job | null>({noop: true}, null);
-    const retrieveIdRef = useRef<HTMLInputElement>(null);
+    var retrieveIdRef = useRef<HTMLInputElement>(null);
 
-    const statusUpdateIdRef = useRef<HTMLInputElement>(null);
-    const statusUpdateStateRef = useRef<HTMLSelectElement>(null);
-    const statusUpdateRef = useRef<HTMLTextAreaElement>(null);
+    var statusUpdateIdRef = useRef<HTMLInputElement>(null);
+    var statusUpdateStateRef = useRef<HTMLSelectElement>(null);
+    var statusUpdateRef = useRef<HTMLTextAreaElement>(null);
```

Comment:
I know it's merged, but any reason for this being changed to `var` instead of `const`?

Reply:
No. Leftover from another attempt. 
Will change back to const